### ISSUE TITLE
[MainUI] remove Basic UI from Setup Wizard

### ISSUE THREAD

**Issue by egoist6** [openhab/openhab-webui#2080]
## The problem
The install wizard suggests to install Basic UI. This could be irritating as the wizard is aimed at making installation easier for **new users**. Existing users who might want to set up a clean system will know anyway to install Basic UI if they are still using sitemap.
![image](https://github.com/openhab/openhab-webui/assets/76903043/71ec78ee-781d-4986-aead-d9edb820cf43)

See discussion here:
https://community.openhab.org/t/first-time-user-wizard-for-oh4/149670/27?u=oliver2

## Your suggestion
Remove Basic UI from wizard

**Comment by lolodomo** [comment]
I rather suggest to keep it as sitemap (and Basic UI) is still used and preferred by many many users. That would be really  a shame if sitemap UI was deliberately hidden to new users.

**Comment by mherwege** [comment]
I agree with @lolodomo on this. I am enhancing this wizard so it gives more context about why Basic UI is there. I find that many new users have an easier time to get a quick sitemap based UI up and running then all of the nice features of mainUI.
See https://github.com/openhab/openhab-webui/issues/3836

**Comment by Lamieur** [comment]
> I find that many new users have an easier time to get a quick sitemap based UI up and running then all of the nice features of mainUI

Hello, I'm a new user. I found this issue while searching for instructions of how to set up Basic UI, which has been installed with my OpenHAB and is a blank page telling me to check documention. Its documentation (link to https://www.openhab.org/addons/ui/basic/) is a single page saying it has 3 settings and why you're not supposed to add a slash to the url (why would a new user even think of that?). Searching for "basic ui" from that (uninformative) page finds ONLY a page telling me how to add some local German train schedule to my Basic UI. Nothing else!

Meanwhile I have already set up whatever OpenHAB's main page offers (no idea how you veterans call it) to create tiles with all the sensors and controls I have so far, by clicking through the page.

I was searching for instructions on how to set up the Basic UI (which I already failed to set up right after the installation) because an external hardware vendor showed it and it looked WAY better than my tiles, so I quickly checked if I have it installed, and I do! Except it can't be made to do anything like the screenshots, or anything at all. It has been scrubbed from the internets.

I assume this used to be a supported way to customize your front page, but you can see my frustration in trying to find any documentation of how to set it up. It doesn't exist. If it does, FIRST make sure <openhab-url>/addons/ui/ui-basic links to it.

Otherwise, this is something that the wizard told me to install, then it wasted a bunch of my time with absolutely zero payout.

Please, if you stopped supporting this add-on completely, remove it from new users' sight.

**Comment by JustinGeorgi** [comment]
> Searching for "basic ui" from that (uninformative) page finds ONLY a page telling me how to add some local German train schedule to my Basic UI. Nothing else!

Basic UI is the add-on that gives you sitemaps. There is, in fact, very little configuration for the Basic UI add-on so in the add-on help page there is not much.

Making a sitemap is a different from configuring the add-on and there is, in fact, a lengthy and detailed help page in the main help docs (not the add-on specific docs): https://www.openhab.org/docs/ui/sitemaps.html#sitemaps

**Comment by mherwege** [comment]
Just to second @JustingGeorgi here, rendering sitemaps is not deprecated at all. Sitemaps are an alternative way to define the content of a UI that is, while less flexible then the main UI, a quick an declarative way to define a UI. It is natively supported in the Android and iOS apps and it is the only way to define a UI for Garmin watches. It is maintained and does get functional enhancements. To render a sitemap based UI on the web, you need BasicUI.
I do agree that this is not obvious and clear from the wizard. That's one of the reasons I have been working on improving this wizard and first time setup exeperience. There is an open PR https://github.com/openhab/openhab-webui/pull/3845 that (among other things) adds context to why BasicUI is proposed.

**Comment by lolodomo** [comment]
Even if there is not a lot more to add, I will check this page and at least update the outdated configuration section (only 2 parameters now available), introduce the preferences page and add a link to the sitemap page.
https://www.openhab.org/addons/ui/basic/
